src/scss/Skeleton.scss

```diff
@@ -0,0 +1,26 @@
+.tv-show-poster--skeleton {
+  width: 180px;
+  height: 250px;
+  margin-top: 20px;
+  margin: auto;
+  background: white;
+  padding: 20px;
+  //   border: 1px solid gray;
+}
+
+.tv-show-poster-image--skeleton {
+  background: linear-gradient(-90deg, #f0f0f0 0%, #f9f9f9 50%, #f0f0f0 100%);
```

Comment:
the animation is fairly hard to see on against the white background. Maybe more greyer would be better

Reply:
I will do that in my next push